mobile/mobile_40.asm

```diff
@@ -1584,20 +1585,35 @@ _LinkBattleSendReceiveAction:
 	inc a
 	jr z, .waiting
 
+	vc_hook send_byt2_ret
+	vc_patch send_byt2_wait
+if DEF(_CRYSTALVC)
+	ld b, 26
+else
 	ld b, 10
+endc
+	vc_patch_end
 .receive
 	call DelayFrame
 	call LinkTransfer
 	dec b
 	jr nz, .receive
 
+	vc_hook send_dummy
+	vc_patch send_dummy_wait
+if DEF(_CRYSTALVC)
+	ld b, 26
+else
 	ld b, 10
+endc
+	vc_patch_end
 .acknowledge
 	call DelayFrame
 	call LinkDataReceived
 	dec b
 	jr nz, .acknowledge
 
+.VC_send_dummy_end::
```

Comment:
Do we need this label? Should it be a `vc_hook`?

Reply:
woops... looks like I missed one. Yes it should be `vc_hook send_dummy_end`. I'll fix it when I get home from work.